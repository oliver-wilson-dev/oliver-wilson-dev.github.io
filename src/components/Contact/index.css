.contactIcons {
    line-height: 50px;
    height: 50px;
}

.contactDetail {
    display: inline-block;
    margin: 0 5px 0 auto;
}