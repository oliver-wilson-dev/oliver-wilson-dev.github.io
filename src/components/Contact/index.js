import React from 'react';
import PropTypes from 'prop-types';
import Section from '../Section';
import styles from './index.css';

const Contact = ({ title, phone: phoneNumber, email }) => (
  <Section title={title}>
    <div aria-label={`my phone number is ${phoneNumber}`}>
      <h4 className={styles.contactDetail} aria-hidden>
        Phone:
      </h4>
      <span aria-hidden>{phoneNumber}</span>
    </div>
    <div aria-label={`my email address is ${email}`}>
      <h4 className={styles.contactDetail} aria-hidden>
        Email:
      </h4>
      <span aria-hidden>{email}</span>
    </div>
  </Section>
);

Contact.propTypes = {
  title: PropTypes.string.isRequired,
  phone: PropTypes.string.isRequired,
  email: PropTypes.string.isRequired
};

export default Contact;
