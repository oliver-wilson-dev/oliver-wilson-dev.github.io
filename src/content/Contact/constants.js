export const EMAIL = 'oliver.wilson@asos.com';
export const PHONE_NUMBER = '07580081310';
