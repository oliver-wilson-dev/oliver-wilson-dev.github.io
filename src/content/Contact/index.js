import React from 'react';
import styles from '../index.css';
import { EMAIL, PHONE_NUMBER } from './constants';

export default {
  title: 'Get in touch',
  content:
  <div>
    <p>
      <h4 className={styles.contactDetail}>
        Phone:
      </h4>
      <span>{PHONE_NUMBER}</span>
    </p>
    <p>
      <h4 className={styles.contactDetail}>
        Email:
      </h4>
      <span>{EMAIL}</span>
    </p>
  </div>,
};
