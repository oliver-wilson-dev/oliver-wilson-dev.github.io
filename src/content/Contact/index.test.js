import Contact from './index';

describe('Contact Section', () => {
  it('should render correctly', () => {
    expect(Contact).toMatchSnapshot();
  });
});
