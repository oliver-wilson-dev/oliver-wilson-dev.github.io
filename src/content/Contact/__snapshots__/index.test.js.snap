// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact Section should render correctly 1`] = `
Object {
  "content": <div>
    <p>
      <h4
        className="contactDetail"
      >
        Phone:
      </h4>
      <span>
        07580081310
      </span>
    </p>
    <p>
      <h4
        className="contactDetail"
      >
        Email:
      </h4>
      <span>
        oliver.wilson@asos.com
      </span>
    </p>
  </div>,
  "title": "Get in touch",
}
`;
